### PR TITLE
Added contains to Column

### PR DESCRIPTION
**Pull request by Braamling**
The [contains](https://spark.apache.org/docs/2.4.4/api/python/pyspark.sql.html#pyspark.sql.Column.contains) method is missing from the stubs causing mypy to raise `error: "Column" not callable`.

This PR adds the typehints to 2.4 specifically (the version we are using), but it should probably also be added to the other versions.

### CODE REVIEW

**Comment by Braamling** [comment]
@zero323 would you be able to take a look?

File: third_party/3/pyspark/sql/column.pyi [hunk 1]
```diff
@@ -81,6 +81,7 @@ class Column:
     def alias(self, *alias: str, **kwargs: Any) -> Column: ...
     def name(self, *alias: str) -> Column: ...
     def cast(self, dataType: Union[DataType, str]) -> Column: ...
+    def contains(self, other: Union[DataType, str]) -> Column: ...
```
Comment (zero323) [inline third_party/3/pyspark/sql/column.pyi]: I'd prefer to keep this consistent with [upstream](https://github.com/apache/spark/blob/94bd480761b9f0cf8dc5b87421a4f63480d8e3a9/python/pyspark/sql/column.pyi#L118) and have `Any` here (see https://github.com/apache/spark/pull/31823#pullrequestreview-611607868)

**Comment by zero323** [comment]
Thank you for your proposal and sorry for a delayed response ‒ I am not much around here these days.

**Comment by Braamling** [comment]
Thanks @zero323, I've changed the type to `Any`. Does this look better?

**Review by zero323 — APPROVED** [review]

**Comment by zero323** [comment]
Thanks @Braamling Merged into branch-2.4 and branch-3.0

**Comment by Braamling** [comment]
@zero323 would it be possible to deploy this fix into the stubs for 2.4? We frequently run into this issue and would like to use the updates stubs :)

**Comment by zero323** [comment]
Not a problem @Braamling 

Deployed to PyPi: 

- [3.0.0.post2](https://pypi.org/project/pyspark-stubs/3.0.0.post2/)
- [2.4.0.post10](https://pypi.org/project/pyspark-stubs/2.4.0.post10/)

I'll handle conda forge once bot prepares the PR.

**Pull request merged**